frontend/src/main/scala/akkaviz/frontend/FrontendApp.scala

```diff
@@ -28,6 +28,8 @@ object FrontendApp extends JSApp with Persistence
 
   private val repo = new ActorRepository()
 
+  val MaxThroughputLogLen = 30
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2145884959/issues/source?bid=3121282&fileBranchId=3151611#l31)